source/viz.md

```diff
@@ -805,23 +848,23 @@ you can clearly see the mutated output from the table.
 ```
 
 ```{code-cell} ipython3
+canadian_population = 35_151_728
 can_lang = can_lang.assign(
-    mother_tongue_percent=(can_lang['mother_tongue']/35151728) * 100,
-    most_at_home_percent=(can_lang['most_at_home']/35151728) * 100
+    mother_tongue_percent=(can_lang['mother_tongue'] / canadian_population) * 100,
+    most_at_home_percent=(can_lang['most_at_home'] / canadian_population) * 100
 )
 can_lang[['mother_tongue_percent', 'most_at_home_percent']]
-
 ```
 
-Finally, we will edit the visualization to use the percentages we just computed
+Next, we will edit the visualization to use the percentages we just computed
 (and change our axis labels to reflect this change in
 units). {numref}`can_lang_plot_percent` displays
 the final result.
-
-
+Here all the tick labels fit by default so we are not changing the labels to include suffixes
+(note that suffixes can also be harder to understand for small quantities,
```

Comment:
separate long parenthetical out into its own sentence, also edit the new sentence to shorten it a bit for clarity